core/client/hdfs/src/main/java/alluxio/hadoop/HadoopUtils.java

```diff
@@ -140,7 +140,7 @@ public static String toStringHadoopInputSplit(InputSplit is) {
    * System properties, and they're not already set.
    *
    * This function is duplicated from {@code alluxio.underfs.hdfs.HdfsUnderFileSystemUtils}, to
-   * prevent the module alluxio-core-client from depending on the module alluxio-underfs.
+   * prevent the module alluxio-core-client-hdfs from depending on the module alluxio-underfs.
```

Comment:
should we move the package from `alluxio.hadoop` to `alluxio.client.hdfs` (in here maybe also `alluxio.client.fs`, `alluxio.client.keyvalue`)? Or keep the source-code level bw compatibility is still the goal?